scripts/ci/lib.sh

```diff
@@ -776,18 +776,13 @@ END
             ;;
         *)
             cat >> "${image_list}" << END
+central-db ${tag}
 main ${tag}
 roxctl ${tag}
 END
             ;;
     esac
 
```

Comment:
Due to removing this can you also add `central-db ${tag}` to the `*-race-condition-qa-e2e-tests)` case?